gammapy/irf/tests/test_psf_analytical.py

```diff
@@ -35,3 +45,25 @@ def test_EnergyDependentMultiGaussPSF_write(tmpdir):
     # see e.g. https://travis-ci.org/gammapy/gammapy/jobs/31056341#L1162
     # assert hdu_list[1].verify_checksum() == 1
     assert len(hdu_list) == 2
+
+
+@requires_dependency('scipy')
+@requires_data('gammapy-extra')
+@pytest.mark.parametrize(('energy'), ENERGIES)
+def test_to_table_psf(energy):
+    filename = gammapy_extra.filename('test_datasets/unbundled/irfs/psf.fits')
```

Comment:
The files in `test_datasets/unbundled` are old and we should get rid of them.
Please use `datasets/hess-crab4-hd-hap-prod2/run023400-023599/run023523/hess_psf_3gauss_023523.fits.gz` here which matches the files we produce now.
Probably this cleanup will only happen when the HESS public data release is through A&R, i.e files are final.
But still ... let's test the latest files we have for now.